src/render.rs

```diff
@@ -141,11 +142,31 @@ impl<'a> Default for MarkdownRenderer<'a> {
 /// let text = "[Rust](https://rust-lang.org/) is an awesome *systems programming* language!";
 /// let rendered = markdown_to_html(text)?;
 /// ```
-pub fn markdown_to_html(text: &str) -> CargoResult<String> {
+fn markdown_to_html(text: &str) -> CargoResult<String> {
     let renderer = MarkdownRenderer::new();
     renderer.to_html(text)
 }
 
+static MARKDOWN_EXTENSIONS: [&'static str; 7] = [
```

Comment:
Even though this is not public & pretty self-explanatory, we might want to add a small doc comment to this constant.

Reply:
+1